pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -53,6 +53,12 @@ Before running the following command, ensure that your `dkp` configuration **ref
     dkp upgrade kommander --charts-bundle dkp-kommander-charts-bundle-v.2.2.0.tar.gz --kommander-applications-repository kommander-applications_v.2.2.0.tar.gz
     ```
 
+    For air-gapped **with** DKP Catalog Applications in a multi-cluster environment:
+
+    ```bash
+    dkp upgrade kommander --charts-bundle dkp-kommander-charts-bundle-v.2.2.0.tar.gz --charts-bundle dkp-catalog-applications-charts-bundle-v2.2.0.tar.gz --kommander-applications-repository kommander-applications_v.2.2.0.tar.gz
```

Comment:
```suggestion
    dkp upgrade kommander --charts-bundle dkp-kommander-charts-bundle-v2.2.0.tar.gz --charts-bundle dkp-catalog-applications-charts-bundle-v2.2.0.tar.gz --kommander-applications-repository kommander-applications_v2.2.0.tar.gz
```

Reply:
🔍  fixed in 86e40871e1f7357357d3cf56ca5c6d23b07c2d82!